effectful-core/src/Effectful/Exception.hs

```diff
@@ -1,87 +1,112 @@
--- | The 'Eff' monad comes with instances of 'MonadThrow', 'MonadCatch' and
--- 'MonadMask' from the
--- [@exceptions@](https://hackage.haskell.org/package/exceptions) library, so
--- this module simply re-exports the interface of the
--- [@safe-exceptions@](https://hackage.haskell.org/package/safe-exceptions)
--- library.
---
--- Why @safe-exceptions@ and not @exceptions@? Because the former makes it much
--- easier to correctly deal with asynchronous exceptions (for more information
--- see its [README](https://github.com/fpco/safe-exceptions#readme)) and
--- provides more convenience functions.
+{-# LANGUAGE CPP #-}
+-- | Support for runtime exceptions.
+--
+-- The 'Eff' monad provides instances of 'C.MonadThrow', 'C.MonadCatch' and
+-- 'C.MonadMask', so any function that works with them can be used
+-- seamlessly. This includes API of the following libraries:
```

Comment:
```suggestion
-- seamlessly. This modules provides an API similar to those of the following libraries:
```

"includes" might be a bit to strong here; Maybe we should go with a phrasing that is a bit more fuzzier.

Reply:
I shuffled the top section, should be good now.